utils/api/requests.js

```diff
@@ -222,6 +222,45 @@ export const createRequest = async ({ dynamicFormData, wareID, accessToken }) =>
   /* eslint-enable camelcase */
 }
 
+export const acceptSOW = ({ request, sowID, accessToken }) => {
+  /* eslint-disable camelcase */
+  const sow = {
+    name: request.title,
```

Comment:
can this be refactored so that the common properties between this object and `pg_quote_group` in "createRequest" are put into their own function? then called upon by "createRequest" and "acceptSOW"?

a la "configureDocuments", "configureSOWs" and "configurePOs".